libc/src/__support/CPP/atomic.h

```diff
@@ -62,6 +62,18 @@ template <typename T> struct Atomic {
     return static_cast<int>(mem_scope);
   }
 
+  LIBC_INLINE static T *addressof(T &ref) { return __builtin_addressof(ref); }
```

Comment:
Do we need `static`, here and below?

Reply:
`constexpr` can only be static, otherwise the compiler would complain.
The function here can be static or not but it does not depend on the member field so `static` seems more suitable.